packages/openid4vc/src/shared/utils.ts

```diff
@@ -165,6 +247,16 @@ export async function openIdTokenIssuerToJwtIssuer(
     }
   }
 
+  if (openId4VcTokenIssuer.method === 'openid-federation') {
+    // TODO: Not sure what we want here if we need to add a new type to the sphereon library or that we can do it with the custom issuer
```

Comment:
let's add a new type

Reply:
or be more explicit that this is about federation (but new type has my pref)

Reply:
Looked into changing the type in the package of spereon but that took a bit more time. So I think it's better to do it after this funke round.